Added more examples and other improvements in the doc (#2)
code-rhapsodie/dataflow-bundle: README.md

}
```

> You can set up this reader like that:

Reviewer: ```suggestion
You can set up this reader like this:
```

> You can set up this reader like that:

Reviewer: ```suggestion
You can set up this reader as follows:
```
